perun-base/src/main/resources/perun-roles.yml

```diff
@@ -4413,14 +4413,6 @@ perun_policies:
     include_policies:
       - default_policy
 
-  getTask_int_int_policy:
```

Comment:
Just a note if you are changing authorization - be sure to check this with the GUI team. I know you didn't change outer API, but generally I can't tell, if they are not using any of these authorization policies in GUI to decide, whether to show/hide users particular functionality.

Reply:
I am not sure what you mean. I did not change any authorization policy, just removed unused methods and their entries in policy. Can you clarify, please?

Reply:
In GUI, to resolve if some user can call a certain method, they use some of these policies (which are accessible via RPC API). They have to hardcore the names of these policies. So for example, if they would have used the policy `getTaskResultsForGUIByTaskOnlyNewest_int_policy` to determine if a user is allowed to call the method `getTaskResultsForGUIByTaskOnlyNewest` it would not work anymore, since this method now uses the `getTaskResultsByTaskOnlyNewest_int_policy` policy.

However, I have searched the GUI repository, and I don't think they are using any of the policies, you have removed.

Reply:
Hello guys,
@Vojtech-Sassmann well we discovered in hard way thats not true. We are currently using "getTaskResultsForGUIByTask_int_policy" policy.
What I dont understand is - in TasksManagerMethod there is method "getTaskResultsForGUIByTask" that gui is currently  calling, so I would assume there is policy that would be named the same. So now I dont know what policy I should use.
Usually gui(me) is oriented what is in the *ManagerMethod.